Make script also replace . with nothing to permit input like "7.0 7.5 8.0"
--- CMakeLists.txt.orig	2021-10-01 20:01:58.357661378 +0200
+++ CMakeLists.txt	2021-10-01 20:02:35.524562185 +0200
@@ -263,6 +263,7 @@
 
 #replace ' ' with ; to match the proper cmake format
 STRING(REGEX REPLACE " " ";" CUDA_ARCH ${CUDA_ARCH})
+STRING(REGEX REPLACE "." "" CUDA_ARCH ${CUDA_ARCH})
 
 #set the compiler flags for each NV target
 FOREACH(target ${CUDA_ARCH})
